Directory.Build.targets

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <!-- Track compiler separately from Arcade.-->
     <PackageReference Include="Microsoft.Net.Compilers.Toolset"
-        Version="$(MicrosoftNetCompilersToolsetVersion)"
+        Version="$(MicrosoftNetCompilersToolsetPackageVersion)"
```

Comment:
I couldn't get `darc` to accept `MicrosoftNetCompilersToolsetPackageVersion` when I tried in EF Core and it updated `Version.props` correctly using `MicrosoftNetCompilersToolsetVersion`. But, I agree `MicrosoftNetCompilersToolsetVersion` looked weird. Glad to see this fixed.